### ISSUE TITLE
NVDA reader does not announce the dialog title and content upon Alert Dialog open.

### ISSUE THREAD

**Issue by rulyu** [nvaccess/nvda#10186]
### Steps to reproduce:
Go to the example [alertdialog example on the W3C Standards site](https://www.w3.org/TR/wai-aria-practices/examples/dialog-modal/alertdialog.html),
click on the "save" or "discard" button to trigger alert dialog open.
Listen to what nvda reader reads for the dialog. 
![image](https://user-images.githubusercontent.com/55103654/64635871-68110d00-d3b5-11e9-8b02-1fe789d2d7cb.png)

### Actual behavior:
The nvda reader only reads the "no" button in the dialog, does not read the title and the content of the dialog.
### Expected behavior:
NVDA reader should read the title and the content of the dialog.

Narrator on Edge reads out the title and content of the dialog correctly. NVDA on Chrome, however, does not
#### NVDA installed/portable/running from source:
installed on Windows
#### NVDA version:
NVDA version 2019.2

#### Windows version:
Windows 10
#### Name and version of other software in use when reproducing the issue:
Chrome version 76
#### Does the issue still occur after restarting your PC?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No.

**Comment by Brian1Gaff** [comment]
It worked in Waterfox just now.
on the latest Alpha.


bglists@blueyonder.co.uk 
Sent via blueyonder. 
Please address personal E-mail to:-
briang1@blueyonder.co.uk, putting 'Brian Gaff'
in the display name field.
Newsgroup monitored: alt.comp.blind-users

**Comment by emilyuhde** [comment]
I'm running into this issue with NVDA as well when `role="alertdialog"` and the content text is being pointed to with `aria-describedby`. It would be great if this issue could be addressed soon.

**Comment by Adriani90** [comment]
This is still reproducible in NVDA 2019.3.1 and Chrome 80.
cc: @feerrenrut

**Comment by Adriani90** [comment]
Closing as duplicate of #9368.
